src/main/resources/anniv5Resources/localization/eng/pickthemallpack/Cardstrings.json

```diff
@@ -0,0 +1,54 @@
+{
+  "${ModID}:BloodWall": {
+    "NAME": "BloodWall",
+    "DESCRIPTION": "Gain !B! Block. NL Gain !M! stslib:Temporary_HP.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Lose !${ModID}:m2! HP."]
+  },
+  "${ModID}:GoldenPick": {
+    "NAME": "Golden Pick",
+    "DESCRIPTION": "Deal !D! damage.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Gain !M! *Gold plus !${ModID}:m2! *Gold for each card in your deck."]
+  },
+  "${ModID}:DeckCheck": {
+    "NAME": "Deck Check",
+    "DESCRIPTION": "Deal damage equal to the larger of your draw or discard pile sizes.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Gain the other reward cards with a ${ModID}:randomized gem.", " NL ${ModID}:Pickup: Gain the other reward cards with a {0}.", " NL (Deals !D! damage.)"]
+  },
+  "${ModID}:BigBoots": {
+    "NAME": "Big Boots",
+    "DESCRIPTION": "If your draw or discard pile has !${ModID}:m2!+ cards, draw !M! cards. NL Draw 1 card.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Add a *Stack to your deck."]
+  },
+  "${ModID}:PackGear": {
+    "NAME": "Pack Gear",
+    "DESCRIPTION": "Gain 1 ${ModID}:Metallicize for every !M! cards in your draw and discard piles combined (rounded up).",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Add an *Aggregate to your deck.", " NL (Gains !${ModID}:m2! *Metallicize.)"]
+  },
+  "${ModID}:TouchedByFlame": {
+    "NAME": "Touched by Flame",
+    "DESCRIPTION": "Apply !M! ${ModID}:Ignite. NL Apply !${ModID}:m2! Weak. NL Apply !${ModID}:m2! Vulnerable.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Lose {0} Max HP."]
+  },
+  "${ModID}:TreasureChest": {
+    "NAME": "Treasure Chest",
+    "DESCRIPTION": "Exhaust. NL Remove this from your deck after playing it !M! times.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup: Gain a random common relic."]
+  },
+  "${ModID}:GrabAndGo": {
+    "NAME": "Grab and Go",
+    "DESCRIPTION": "Unplayable. Innate.",
+    "UPGRADE_DESCRIPTION": "Unplayable.",
+    "EXTENDED_DESCRIPTION": [" NL ${ModID}:Pickup/Removed: Gain/lose a *Question Card. NL Remove this from your deck after the act boss.", "Unplayable. NL Innate."]
```

Comment:
This wording is the best way I found to avoid very long card text.

Pickup/Removed is a variant of the pickup keyword that I added (in the main packmaster project, since that's where the keyword is defined, as part of the Bellord pack).